documentation/projects/proposals/rekognition_data/20240510-implementation_plan_machine_generated_tags_frontend.md

```diff
@@ -0,0 +1,196 @@
+# 2024-05-08 Implementation Plan: Machine-generated tags in the frontend
+
+**Author**: @obulat
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @zackkrida
+- [ ] @AetherUnbound
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/431)
+- [Project Proposal](/projects/proposals/rekognition_data/20240320-project_proposal_rekognition_data.md)
+
+## Overview
+
+<!-- An overview of the implementation plan, if necessary. Save any specific steps for the section(s) below. -->
+
+This implementation plan will describe how we plan to display the
+machine-generated tags separately from the source tags on the single result
+page. All tags will be clickable and lead to a collection view by the selected
+tag. The tag collection view will not be affected as we will not show different
+results for a machine-generated tag than the source tag.
+
+## Expected Outcomes
+
+<!-- List any succinct expected products from this implementation plan. -->
+
+When this work is completed, the single result page will display
+machine-generated tags separately from the source tags. For machine-generated
+tags, we will also display the tag provider and its accuracy.
```

Comment:
Can someone share an example how this would look and work? I don't fully understand the idea and it sounds like a design change is necessary. Please correct me if I'm wrong.